src/funtest/groovy/steps/persistence.groovy

```diff
@@ -16,6 +16,11 @@ And(~/^a valid (.*) binary for (.*) (.*) hosted at (.*)$/) { String platform, St
     insertCandidateVersionInDb(db, candidate, version, platform, url)
 }
 
+And(~/^a valid (.*) binary for (.*) (.*) from (.*) with algorithm (.*) and checksum (.*)$/) {
+    String platform, String candidate, String version, String url, String algorithm, String checksum ->
+        insertCandidateVersionInDb(db, candidate, version, platform, url, algorithm, checksum)
```

Comment:
Let's talk a bit more about how we should persist this. Please see my comments on the DB migration PR.